src/cube_blog.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+import BlogPost from './components/BlogPost';
+import Paginate from './components/Paginate';
+
+class CubeBlog extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount() {
+    updateBlog();
+  }
+
+  select(nav) {
+    this.setState({ });
+  }
+
+  render() {
+    const {pages, posts, canEdit} = this.props
+    return (
+      <>
+        {(pages && pages.length > 1) &&
+          <Paginate pages={pages} />
+        }
+        {posts.map(post =>
+          <BlogPost post={post} canEdit={canEdit}/>
```

Comment:
these should have `key`s i think

Reply:
I don't understand, can you elaborate?

Reply:
any elements in an array (like `BlogPost`) should have a `key` attribute to uniquely identify the element, i'm not positive what the data of `post` is (just `blogSchema?`) but something unique